static/js/components/utils/AutocompleteSettings.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
Can you rename `AutocompleteSettings` to `AutoCompleteSettings` for consistency?